docs/pages/identity-security/access-graph/access-graph.mdx

```diff
@@ -1,14 +1,23 @@
 ---
-title: "Self-Hosting Teleport Access Graph"
+title: "Self-Hosting Teleport Identity Security"
+sidebar_label: Self-Host Teleport Identity Security
 description: Explains how to deploy Access Graph alongside a self-hosted Teleport cluster.
 tags:
  - identity-security
  - access-risks
+sidebar_position: 4
 ---
 
-If you run a self-hosted Teleport cluster, using Access Graph (part of
-Teleport Identity Security) requires running the Access Graph Service on your own
-infrastructure. The following guides show you how to deploy the Access Graph
-Service.
+If you run a self-hosted Teleport cluster, using Teleport Identity Security
+requires running two services on your own infrastructure:
+
+- **Teleport Access Graph Service:** Powers access path exploration features,
+  including the Dashboard, Browse, Crown Jewels, Graph Explorer, and SQL Editor
+  views.
+- **Teleport Identity Activity Center:** Powers audit log centralization and
+  querying, including the Browse and Alerts views.
```

Comment:
@ptgott These are not two separate services you need to run. Identity Activity Center is enabled via configuration in the same service that provides Access Graph. This reads a bit confusing making it seem like you have to run 2 things.